app/mailers/publisher_mailer.rb

```diff
@@ -185,9 +185,9 @@ def unverified_domain_reached_threshold_internal(domain, email)
     )
   end
 
-  def verified_invalid_wallet(publisher, params)
```

Comment:
These `params` were being sent via the soon to be deprecated `PublisherNotifier`, which was used by eyeshade to send these emails when eyeshade was in charge of payout report generation.  The `params` don't ever seem to be used, so I'm removing them entirely.